javascript/javascript1/week3/homework/main-5.js

```diff
@@ -0,0 +1,71 @@
+//CactusIO-interactive (Smart phone usage app)
+
+//ADDING AN ACTIVITY
+const activities = [];
+function addActivity(activity, duration) {
+  const activityNote = new Object();
+  let date = new Date();
+  activityNote.date = date.toLocaleDateString(`en-US`);
+  activityNote.activity = activity;
+  activityNote.duration = duration;
+  activities.push(activityNote);
+}
+
+addActivity(`Youtube`, 30);
+addActivity(`Facebook`, 20);
+addActivity(`Instagram`, 45);
+
+activities.push({ date: `12/27/2021`, activity: `Youtube`, duration: 30 });
+activities.push({ date: `12/29/2021`, activity: `Instagram`, duration: 40 });
+activities.push({ date: `12/29/2021`, activity: `Youtube`, duration: 25 });
+activities.push({ date: `12/29/2021`, activity: `Facebook`, duration: 40 });
+
+console.log(activities);
+
+//SHOWING STATUS OF SCREEN TIME ON A CERTAIN DATE
+function showStatus(date) {
+  let totalDurationStatus = 0;
+  let numberActivities = 0;
+  const limit = 100;
+  for (i = 0; i < activities.length; i++) {
+    if (activities[i].date == date) {
```

Comment:
Try to use the triple equal("===") as a comparison operator, if the types a different is always better to parse number to string or vice-versa than using the double equal ("=="), it won't have an effect here but it is a good practice to have 😊

Reply:
Thanks for the great advice:) P.S. I usually use triple equal but why I did double in this case I cannot explain. I finished this exercise more than 2 weeks ago.